doc/source/refs.rst

```diff
@@ -72,4 +72,7 @@ References
    *Introduction to Fourier Analysis on Euclidean Spaces*.
    Princeton University Press, 1971.
 
+.. [Val2014] Valkonen, Teukolsky
```

Comment:
Author format lacks first names, compare rest of file (can you remove the extra dot at the Rockafellar citation?).